client/src/planwise/client/projects2/handlers.cljs

```diff
@@ -11,6 +11,12 @@
 
 (def in-projects2 (rf/path [:projects2]))
 
+(defn- reset-state
```

Comment:
This name is too generic. What do you think about `clear-current-project`?

Reply:
It seems more appropiate.